tasks/compile.rb

```diff
@@ -0,0 +1,36 @@
+require "compile_to_file"
+
+desc "Compile the application to a single file for deployment"
+task :compile do
+  CompileToFile::App.new do |app|
+
+    app.skip_requires_for("automate_liveness_agent")
+
+    app.add_lib_files(%w{
```

Comment:
What's the benefit of explicitly specifying each file instead of just globing the `lib` directory?